.github/workflows/main.yml

```diff
@@ -125,7 +122,7 @@ jobs:
       matrix:
         toolchain:
           - stable
-          - 1.51.0
+          - 1.56.0
```

Comment:
I see. Should lines 102 and 125 also be updated to 1.56.1 then as well (instead of 1.56.0)?